io.github.shadowmario.fnf-psychengine.yml

```diff
@@ -0,0 +1,87 @@
+app-id: io.github.shadowmario.fnf-psychengine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: io.github.shadowmario.fnf-psychengine.sh
+
+finish-args:
+  # gamepad support
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+
+modules:
+  - name: fnf-psychengine
+    modules:
+      - name: gamemode
+        buildsystem: meson
+        config-opts:
+          - -Dwith-sd-bus-provider=no-daemon
+          - -Dwith-examples=false
+        sources:
+          - type: git
+            url: https://github.com/FeralInteractive/gamemode
+            tag: "1.8.1"
+            x-checker-data:
+            type: git
+            tag-pattern: ^([\\d.]+)$
+    sources:
+      - type: archive
+        url: https://github.com/HaxeFoundation/neko/releases/download/v2-3-0/neko-2.3.0-linux64.tar.gz
+        sha256: 26dda28d0a51407f26218ba9c2c355c8eb23cf2b0b617274b00e4b9170fe69eb
+        dest: haxe/
+        x-checker-data:
+          type: anitya
+          project-id: 10885
+          url-template: https://github.com/HaxeFoundation/neko/releases/download/v$version/neko-$version-linux64.tar.gz
+      - type: archive
+        url: https://github.com/HaxeFoundation/haxe/releases/download/4.3.3/haxe-4.3.3-linux64.tar.gz
```

Comment:
so how will this work on aarch64?

Reply:
Fair call, I didn't think of that. I'll see if I can get Haxe and NekoVM built from source as they don't supply arm64 binaries; otherwise I may just have to limit this to x86 using `only-arch`.

Reply:
I have indeed limited the build to x86_64 now - NekoVM and Haxe come with ridiculously large dependency chains including ocaml, Apache, MariaDB and various others, so setting this up would inflate the builds to absurd proportions here.

Is there documentation on creating new SDKs? I might be up for creating ocaml and, based on that, Haxe SDK Flatpaks as I know a nontrivial number of game developers who work with Haxe who would find this useful...